### PR TITLE
More sensible 'days ago' terminology

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ const formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+const inSeconds = {
+	hour: 60 * 60,
+	day: 24 * 60 * 60,
+	week: 7 * 24 * 60 * 60,
+	month: (365 * 24 * 60 * 60) / 12,
+	year: 365 * 24 * 60 * 60
+}
```
Comment (i-like-robots) [inline main.js]: colon police

Comment (ironsidevsquincy) [inline main.js]: sounds painful

File: main.js [hunk 2]
```diff
@@ -34,6 +34,14 @@ const formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+const inSeconds = {
+	hour: 60 * 60,
```
Comment (onishiweb) [inline main.js]: Any reason for not adding `minute` to the `inSeconds` object? Seems odd to leave it out considering the rest of the changes made.

**Comment by onishiweb** [comment]
Looks good. Just the one comment but it should be fine. 

Circle test passes now which is good, the Travis tests should be getting ignored, so @j-mes is going to look into that.

**Comment by j-mes** [comment]
I've removed Travis from this repo, and the next time someone does a PR or commit, it should not have Travis to run the tests.

File: main.js (hunk 2 above)
Comment (ironsidevsquincy) [inline main.js]: Nope, just an omission

**Comment by ironsidevsquincy** [comment]
Thanks chaps. Comment addressed. Not sure of the process of releasing now, do I merge/you/other?

**Comment by onishiweb** [comment]
Nice one, thanks. 

I think it's probably on us to take care of the merge/release, so I'll do that now.

**Pull request merged**